.ci/pytorch/test.sh

```diff
@@ -194,13 +194,13 @@ test_python_shard() {
     exit 1
   fi
 
-  time python test/run_test.py --exclude-jit-executor --exclude-distributed-tests --shard "$1" "$NUM_TEST_SHARDS" --verbose
+  time python test/run_test.py --continue-through-error --exclude-jit-executor --exclude-distributed-tests --shard "$1" "$NUM_TEST_SHARDS" --verbose
 
   assert_git_not_dirty
 }
 
 test_python() {
-  time python test/run_test.py --exclude-jit-executor --exclude-distributed-tests --verbose
+  time python test/run_test.py --continue-through-error --exclude-jit-executor --exclude-distributed-tests --verbose
```

Comment:
```suggestion
  time python test/run_test.py --exclude-jit-executor --exclude-distributed-tests --verbose
```

Reply:
Removed `--continue-through-error` in https://github.com/pytorch/pytorch/pull/97988/commits/34795f70287defe82b5bc1198ee303a638049c18